Code-Refactorng:

LeonidVolohov opened the pull request:
Branch for code refactoring and small changes

LeonidVolohov committed:
Code-Refactorng:
 - Added array to spinner directly through xml file in activity_currency.xml and activity_crypto.xml files
 - Removed adapters in CurrencyMain.kt and CryptoСurrencyMain.kt due to lack of need for it
 - Added multiplier for primary rates in currency screen

LeonidVolohov committed:
Code-Refactoring:
 - Changed ImageButton to ImageView in activity_currency.xml file due to the fact that I made a mistake at the beginning when selecting an element for icons and changed icons size
  - Created custom class for chart and moved all the logic in it from CurrencyMain.kt to StockLineChart.kt

kramarov-evg reviewed (approved):
Will leave us with merge-conflicts in at least manifest, but they seem to be easily resolvable.
Thanks for separating charting away from activity :+1:

owlerxan reviewed (approved):
Ok

Commit:
Calculator added to crypto

Commit:
Minor additions for ui

LeonidVolohov committed:
Merge branches 'code-refactoring' and 'crypto-fragments' of https://github.com/LeonidVolohov/Stonks into code-refactoring

 Conflicts:
	app/src/main/AndroidManifest.xml
	app/src/main/java/com/stonks/ui/cryptocurrency/CryptoСurrencyMain.kt

kramarov-evg committed:
Created layout file for stocks fragment

Changes:
- Added needed strings, dimensions and icons
- Created basic layout
- Tiny code clean-up

kramarov-evg committed:
Created most basic fragment implementation

Changes:
- Added new class for stocks fragment
- Fixes to layout code-style

kramarov-evg committed:
Fixed strings.xml

Changes:
- Fixed style of stocks array

LeonidVolohov committed:
Merge branches 'code-refactoring' and 'stocks_fragment' of https://github.com/LeonidVolohov/Stonks into code-refactoring
